### ISSUE TITLE
Strange getopt version in nixpkgs unstable

### ISSUE THREAD

**Issue by kraileth** [repology/repology#643]
The nixpkgs unstable repo holds a package of getopt that reports as "1003.1-2008". This is clearly garbage and makes it look like the correct version 1.1.6 is outdated.

**Comment by AMDmi3** [comment]
I've seen the report. There's no need to duplicate it as an issue.

**Comment by kraileth** [comment]
Sorry, I didn't find anything on this problem. Would repology-rules been the right place? And how are problems like this reported correctly instead of opening an issue?

**Comment by AMDmi3** [comment]
> Would repology-rules been the right place?

Yes.

> Sorry, I didn't find anything on this problem.
> And how are problems like this reported correctly instead of opening an issue?

There is **report** button for any metapackage:

https://repology.org/metapackage/libdmx/report
https://repology.org/metapackage/getopt/report

I got the reports for these two packages the same time I've got the two issues for the very same packages from you, so I though you've submitted both the issues and the reports. Sorry if I was wrong.
